src/python/pants/backend/awslambda/common/awslambda_common_rules.py

```diff
@@ -23,7 +22,7 @@ class AWSLambdaError(Exception):
 @dataclass(frozen=True)
 class CreatedAWSLambda:
     digest: Digest
-    name: str
+    zip_file_relpath: str
```

Comment:
According to AWS Lambda docs, a "deployment package" is a zip file: https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-features.html#gettingstarted-features-package.